### DISCUSSION TITLE
Buttons to switch spaces

### DISCUSSION THREAD

**Discussion by pingoklooi** [lawand-dot-io/taskbar#52] [Feature requests]
Buttons to switch desktops. E.g. button 1 = desktop 2, button 2 = desktop 2, so forth and so on.

**Comment by socalengineer** [comment]
You can do this already with keyboard shortcuts in mission control. See my screenshot.
<img width="723" height="874" alt="Screenshot 2026-02-01 at 7 13 24 PM" src="https://github.com/user-attachments/assets/0b0f1eca-78d1-49e1-b279-02c625e4398c" />

**Reply by lawand-dot-io** [reply]
@socalengineer that's a nice tip, thank you for sharing 👍
